projects/angular/src/forms/styles/_properties.forms.scss

```diff
@@ -12,12 +12,14 @@
   @if variables.$clr-use-custom-properties == true {
     :root {
       // Structure
+      --clr-forms-baseline: #{variables.$clr_baselineRem_0_25};
       --clr-form-disabled-background-color: var(--clr-color-neutral-400);
 
       // Generic
       --clr-forms-label-color: var(--clr-color-neutral-800);
       --clr-forms-text-color: var(--clr-color-neutral-1000);
       --clr-forms-invalid-color: var(--clr-color-danger-800);
+      --clr-forms-invalid-text-color: var(--clr-color-danger-800);
```

Comment:
I think we should file a ticket to do this rename in a breaking change PR for v16 and just keep using the exiting CSS custom property instead of duplicating it. The code in this PR uses both of these and it could cause confusion.

Reply:
It's equal for the default theme but different when using core. We can skip adding these to _properties file and it would use the custom property if available or the default value for example?

Reply:
I believe we can go on with this and do a cleanup after the token migration.
We can add and //TODO: ATM comment to keep track of this intent.